Source/Scene/ModelExperimental/GeometryPipelineStage.js

```diff
@@ -530,4 +545,47 @@ function handleBitangents(shaderBuilder, attributes) {
   );
 }
 
+function updateStatistics(renderResources, primitive) {
+  const statistics = renderResources.model.statistics;
+  const indicesCount = renderResources.count;
+  const primitiveType = primitive.primitiveType;
+
+  if (primitiveType === PrimitiveType.POINTS) {
```

Comment:
If we add this, how will this interact with debug wireframe mode?

Reply:
Since wireframe is a debug option I think we would continue to show trianglesLength.

Reply:
Adding `linesLength` is a pretty wide-reaching change if we want it in the 3D Tiles inspector (as we need to update every `Cesium3DTileContent` subclass and other things). I'm going to do this in a follow-up PR since this one is already quite large.

Reply:
Could you open an issue for now? We can get to it post `ModelExperimental`.

Reply:
Opened https://github.com/CesiumGS/cesium/issues/10431 and https://github.com/CesiumGS/cesium/issues/10432